Source/SocketIO/Client/SocketIOClient.swift

```diff
@@ -230,7 +230,7 @@ open class SocketIOClient : NSObject, SocketIOClientSpec {
     /// - parameter event: The event to send.
     /// - parameter items: The items to send with this event. May be left out.
     /// - parameter completion: Callback called on transport write completion.
-    open func emit(_ event: String, _ items: SocketData..., completion: @escaping () -> ())  {
+    open func emit(_ event: String, _ items: SocketData..., completion: (() -> ())? = nil)  {
```

Comment:
I think this one can stay nil right? If you're calling this method then you're explicitly wanting to use the callback feature.

Reply:
You are right. I removed the unnecessary function calls.